src/mlstacks/terraform/aws-modular/output_file.tf

```diff
@@ -76,6 +76,13 @@ resource "local_file" "stack_file" {
         flavor: seldon
         name: eks_seldon_model_deployer
         configuration: {"kubernetes_context": "${aws_eks_cluster.cluster[0].arn}", "kubernetes_namespace": "${local.seldon.workloads_namespace}", "base_url": "http://${module.istio[0].ingress-hostname}:${module.istio[0].ingress-port}"}}
+%{endif}
+%{if var.enable_model_deployer_huggingface}
+      model_deployer:
+        id: ${uuid()}
+        flavor: huggingface
+        name: eks_huggingface_model_deployer
+        configuration: {"kubernetes_context": "${aws_eks_cluster.cluster[0].arn}", "kubernetes_namespace": "${local.huggingface.workloads_namespace}", "base_url": "http://${module.istio[0].ingress-hostname}:${module.istio[0].ingress-port}"}}
```

Comment:
This needs updating as we're no longer using EKS for this etc..

Reply:
@strickvl Thank you for the review! I understood everything else but for this part, Could you please provide more guidance or resources on how to do this part?